pkg/network/ebpf/kprobe_types.go

```diff
@@ -10,6 +10,7 @@ package ebpf
 /*
 #include "./c/pid_fd.h"
 #include "./c/tracer/tracer.h"
+#include "./c/protocols/http2/decoding-defs.h"
```

Comment:
if you add an include, you need to modify the relevant entry in `ninja_cgo_type_files` ,method is `system_probe.py`